### DISCUSSION TITLE
How to draw AUC curve of test set

### DISCUSSION THREAD

**Discussion by whire97612** [pycaret/pycaret#1897] [Q&A] [answered]
The official document provides to draw the AUC curve of the training set. I want to know how to use the model after training to draw the AUC curve of the test set. Evaluate the model

**Comment by whire97612** [comment, marked as answer]
use_train_data: bool, default = False
        When set to true, train data will be used for plots, instead
        of test data.
